<ui version="4.0" >
 <class>frmCorrect</class>
 <widget class="QDialog" name="frmCorrect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>460</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Wyślij poprawione napisy</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/resources.qrc" >
    <normaloff>:/icon/qnapi-48-icon.png</normaloff>:/icon/qnapi-48-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="lbMovieSelect" >
     <property name="text" >
      <string>Wskaż plik z filmem:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="leMovieSelect" />
     </item>
     <item>
      <widget class="QPushButton" name="pbMovieSelect" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../res/resources.qrc" >
         <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbSubtitlesSelect" >
     <property name="text" >
      <string>Wskaż poprawione napisy do powyższego pliku:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="leSubtitlesSelect" />
     </item>
     <item>
      <widget class="QPushButton" name="pbSubtitlesSelect" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../res/resources.qrc" >
         <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbLanguage" >
       <property name="text" >
        <string>&amp;Język napisów:</string>
       </property>
       <property name="buddy" >
        <cstring>cbLanguage</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbLanguage" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text" >
         <string>Polski</string>
        </property>
        <property name="icon" >
         <iconset resource="../res/resources.qrc" >
          <normaloff>:/languages/pl.gif</normaloff>:/languages/pl.gif</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Angielski</string>
        </property>
        <property name="icon" >
         <iconset resource="../res/resources.qrc" >
          <normaloff>:/languages/en.gif</normaloff>:/languages/en.gif</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbComment" >
     <property name="text" >
      <string>&amp;Opis poprawki:</string>
     </property>
     <property name="buddy" >
      <cstring>teComment</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="teComment" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbAction" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Wypełnij wszystkie pola, aby wysłać poprawkę</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;Anuluj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPost" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Wyślij</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>frmCorrect</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>378</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel" >
     <x>477</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
